common/src/main/java/com/wynntils/models/items/annotators/gui/GuildTerritoryUpgradeAnnotator.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.annotators.gui;
+
+import com.wynntils.core.text.StyledText;
+import com.wynntils.handlers.item.GuiItemAnnotator;
+import com.wynntils.handlers.item.ItemAnnotation;
+import com.wynntils.models.items.items.gui.GuildTerritorryUpgradeItem;
+import com.wynntils.models.territories.type.TerritoryUpgrade;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.item.ItemStack;
+
+public class GuildTerritoryUpgradeAnnotator implements GuiItemAnnotator {
```

Comment:
Can you rename this to `TerritoryUpgradeItem` so it matches `TerritoryItem`'s naming?

Reply:
And make sure to reorder it in `ItemModel` so it's still alphabetical :)